charts/cryostat/templates/cookie_secret.yaml

```diff
@@ -0,0 +1,13 @@
+{{- $secret := (lookup "v1" "Secret" .Release.Namespace (printf "%s-cookie-secret" .Release.Name)) -}}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ .Release.Name }}-cookie-secret
+type: Opaque
+{{ if $secret -}}
+data:
+  COOKIE_SECRET: {{ $secret.data.COOKIE_SECRET }}
+{{ else -}}
+stringData:
+  COOKIE_SECRET: {{ randAlphaNum 32 }}
+{{ end }}
```

Comment:
How about keeping the "cryostat.cookieSecret" and reference it here in the `data` field as in other secrets that helm creates?

```go
{{/*
Generate or retrieve a default value for cookieSecret.
*/}}
{{- define "cryostat.cookieSecret" -}}
{{- $secret := (lookup "v1" "Secret" .Release.Namespace (printf "%s-cookie-secret" .Release.Name)) -}}
{{- if $secret -}}
{{/*
   Use the current secret. Do not regenerate.
*/}}
{{- $secret.data.COOKIE_SECRET -}}
{{- else -}}
{{/*
    Generate a new secret.
*/}}
{{- randAlphaNum 32 | b64enc | quote -}}
{{- end }}
{{- end }}
```